packages/components/src/components/post-header/post-header.scss

```diff
@@ -230,8 +230,7 @@
   }
 
   ::slotted(post-mainnavigation),
-  .navigation-footer,
-  .navigation-target-group {
+  .navigation-footer {
```

Comment:
That selector already exists on line 226/227, please merge them. This will also fix the sonar code smell.